multiqc/qualimap/__init__.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from QualiMap """
+
+from __future__ import print_function
+import json
+import logging
+import os
+
+import multiqc
+
+class MultiqcModule(multiqc.BaseMultiqcModule):
+
+    def __init__(self, report):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__()
+
+        # Static variables
+        self.name = "QualiMap"
+        self.anchor = "qualimap"
+        self.intro = '<p><a href="http://qualimap.bioinfo.cipf.es/" target="_blank">QualiMap</a> \
+             is a platform-independent application written in Java and R that provides both \
+             a Graphical User Inteface (GUI) and a command-line interface to facilitate \
```

Comment:
You can probably trim this down a bit if you want - GUI stuff is a bit irrelevant :sleeping: